internals/secrethub/run.go

```diff
@@ -80,6 +80,13 @@ func (cmd *RunCommand) Run() error {
 	}
 	envSources = append(envSources, flagSource)
 
+	if cmd.template == "" {
+		const defaultTemplate = "secrethub-env.yml"
```

Comment:
Consider that the current practices we recommend in the docs is always suffixing the template with `.tpl`. This breaks from that convention. Now it isn't a change to the status quo, but we should be concious of this.

Reply:
@mackenbach We discussed the name last Friday with the team and preferred to use `.yml` to make it clear and intuitive what the format of the file should be.

> And how do you propose we make it consistent with the current practices?

I'd prefer to use the `.yml` extension in the docs as well, as that makes the format self-explanatory in the docs as well.